app/scripts/views/base.js

```diff
@@ -227,13 +227,13 @@ define(function (require, exports, module) {
           // be returned.
           this._context = null;
           this.$el.html(this.renderTemplate(this.template.bind(this)));
+          this.trigger('rendered');
```

Comment:
Is this worthy of definition in a constants file somewhere? Based on it being used in two disparate locations...?

Reply:
By "this", I mean the string constant `'rendered'`, in case that wasn't clear.

Reply:
I'm fine with that if we do it universally, though I'd like to do it outside of this PR. I did a search of `.trigger('`, there are 142 results between the app and tests, none of which use a named constant. Opened #5406 to discuss.